server/scripts/import-tests/testParser.js

```diff
@@ -0,0 +1,423 @@
+const path = require('path');
+const fse = require('fs-extra');
+const {
+  createTestId,
+  createScenarioId,
+  createAssertionId
+} = require('../../services/PopulatedData/locationOfDataId');
+const { convertAssertionPriority } = require('shared');
+const { getAppUrl } = require('./utils');
+const deepPickEqual = require('../../util/deepPickEqual');
+
+/**
+ * Strategies for different test format versions.
+ */
+const testFormatStrategies = {
+  v1: {
+    createTest: createV1Test,
+    createRenderableContent,
+    createRenderedUrls,
+    createScenarios
```

Comment:
nit: continuining to use the v1 identifier in the function names could be pretty useful to not show any "default" preference or priority.